package.json

```diff
@@ -10,12 +10,11 @@
   "scripts": {
     "format:check": "npx prettier --check './**/*.{ts,tsx,js,json}'",
     "format:fix": "npx prettier --write './**/*.{ts,tsx,js,json}'",
-    "openapi:generate-client": "npm run openapi:generate-client -w client",
-    "generate": "npm run openapi:generate-client",
+    "generate": "npm run generate -ws --if-present",
     "clean": "rimraf ./dist && npm run clean -ws --if-present",
     "clean:all": "npm run clean && rimraf ./node_modules ./**/node_modules/",
     "dist": "rimraf ./dist && copyfiles -e 'node_modules/**' entrypoint.sh '**/package.json' '*/dist/**/*' ./dist",
-    "build": "npm run build -ws --if-present",
+    "build": "npm run generate && npm run build -ws --if-present",
```

Comment:
With the CURRENT changes, this is not needed technically because Your PR is not ignoring the files from `hey-api` then this is not needed.

Reply:
That's right. But will be needed with next commit.